lib/internal/errors.js

```diff
@@ -437,7 +442,10 @@ module.exports = {
   getMessage,
   SystemError,
   codes,
-  E // This is exported only to facilitate testing.
+  // These are exported only to facilitate testing.
+  E,
+  get useOriginalName() { return useOriginalName; },
+  set useOriginalName(value) { useOriginalName = value; }
 };
```

Comment:
Instead of adding the `useOriginalName` couldn't you create a new error helper for more Web Platform related errors separate from the `makeNodeErrorWithCode` one?

Reply:
@jdalton I would if we decide to leave the names alone for those errors...but doesn't seem to worth it if it's just to work around the WPT harness and it's only for testing

Reply:
Is it just a test harness thing are is the WPT fail hinting that URL errors should not have custom name's other than those of the builtin errors? If it means `URL` is not spec compliant then it's an issue for Node. If it is unspec'd then a bug should be filed for WPT.

Reply:
See https://github.com/nodejs/node/issues/11299 If you feel strongly about this feel free to reopen that issue or open a new one. If we were to fix that, it has to be semver-major. This PR is just a work around so that we can update the tests without a semver-major change - I would rather not label a bunch of upcoming test update with semver-major and make them a backport burden.